### PR TITLE
Allow ability to override duration of transition in `Appear` component

### PR DESCRIPTION
**Pull request by jlongster**
It would be nice customize the duration of the `Appear` component so I can make weird custom animations!

### CODE REVIEW

File: src/components/appear.js [hunk 1]
```diff
@@ -36,11 +36,12 @@ class Appear extends Component {
   render() {
     const child = React.Children.only(this.props.children);
     const endValue = this.state.active ? 1 : 0;
+    const transitionDuration = this.props.transitionDuration || 300;
```
Comment (waywaaard) [inline src/components/appear.js]: I think the proper way would be to use reacts defaultProps to set the transitionDuration, e.g.

```javascript 
Appear.defaultProps = { 
    transitionDuration : 300
};
```

**Comment by jlongster** [comment]
Responded to review!

File: src/components/appear.js [hunk 2]
```diff
@@ -57,6 +58,10 @@ class Appear extends Component {
   }
 }
 
+Appear.defaultProps = {
+  transitionDuration: 300
+};
+
 Appear.propTypes = {
```
Comment (waywaaard) [inline src/components/appear.js]: `transitionDuration` needs to be added to the propTypes. Travis error:

```
'transitionDuration' is missing in props validation  react/prop-types
```

**Comment by jlongster** [comment]
Sorry for the churn! I'll see if the tests pass this time myself.

**Comment by jlongster** [comment]
Yay they passed! Thanks for the quick review. Let me know if there is anything else.

**Comment by carlospaelinck** [comment]
This looks good to me 
// cc: @kenwheeler

**Pull request merged**